src/api/agents.ts

```diff
@@ -25,8 +25,8 @@ export function createAgentsApi(
         update(id: string, payload: AgentPayload, options?: RequestInit) {
             return client.patch<Agent>(`/${id}`, payload, options);
         },
-        newChat(agentId: string, options?: RequestInit) {
-            return client.post<Chat>(`/${agentId}/chat`, undefined, options);
+        newChat(agentId: string, payload: { persist: boolean }, options?: RequestInit) {
```

Comment:
call it options

Reply:
There already is options in the function signature and it is the actual payload of the request